reference-architecture/aws-ansible/ose-on-aws.py

```diff
@@ -76,9 +76,12 @@
 @click.option('--github-client-secret', help='GitHub OAuth Client Secret')
 @click.option('--github-organization', multiple=True, help='GitHub Organization')
 @click.option('--s3-username',  help='S3 user for registry access')
-@click.option('--deploy-openshift-metrics',  help='Deploy OpenShift Metrics', type=click.Choice(['true', 'false']), default='false')
+@click.option('--openshift-hosted-metrics-deploy',  help='Deploy OpenShift Metrics', type=click.Choice(['true', 'false']), default='false')
+@click.option('--openshift-hosted-logging-deploy',  help='Deploy OpenShift Logging', type=click.Choice(['true', 'false']), default='false')
 @click.option('--openshift-hosted-metrics-storage-volume-size', default='20Gi', help='Size of OptionShift Metrics Persistent Volume',
               show_default=False)
+@click.option('--openshift-hosted-logging-storage-volume-size', default='100Gi', help='Size of OptionShift Logging Persistent Volume',
```

Comment:
100Gi x n-elasticsearch is a big amount of storage, but anyway, if we decide to go for 100Gi we should make this consistent in our deployments (in Azure is 10Gi)